tests/common/include/tests/common/tools/create_toy_geometry.hpp

```diff
@@ -213,9 +213,9 @@ inline void create_barrel_modules(context_t &ctx, volume_type &vol,
     m_centers.reserve(n_phi_bins * n_z_bins);
 
     // prep work
-    // double pi{static_cast<scalar>(M_PI)};
-    double phi_step = 2 * M_PI / (n_phi_bins);
-    double min_phi = -M_PI + 0.5 * phi_step;
+    // scalar pi{static_cast<scalar>(M_PI)};
+    scalar phi_step = 2 * M_PI / (n_phi_bins);
+    scalar min_phi = -M_PI + 0.5 * phi_step;
```

Comment:
Are you not getting narrowing conversion warnings here?

Reply:
No I don't get the warning even when I set the scalar type to float. I am also curious though